core/src/main/scala/chisel3/internal/Builder.scala

```diff
@@ -292,19 +292,27 @@ private[chisel3] trait HasId extends InstanceId {
   private def refName(c: Component): String = _ref match {
     case Some(arg) => arg.fullName(c)
     case None => {
-      val nameGuess = _computeName(None) match {
-        case Some(name) => s": '$name'"
-        case None       => ""
-      }
-      val parentGuess = _parent match {
-        case Some(ViewParent) => s", in module '${reifyParent.pathName}'"
-        case Some(p)          => s", in module '${p.pathName}'"
-        case None             => ""
-      }
-      throwException("You cannot access the .instanceName or .toTarget of non-hardware Data" + nameGuess + parentGuess)
+      throwException(
```

Comment:
I'm not sure why scalafmt changed the breaking on this line... I ran it as described in CONTRIBUTING.md.

Reply:
It is weird but whatever, we let the tool do what it wants 🤷‍♂️